server/api/eventType_handler.py

```diff
@@ -0,0 +1,86 @@
+from flask import jsonify, Blueprint, request
+from model.model import EventTypes
+from config import db
+
+eventType_handler = Blueprint('eventType_handler', __name__)
+
+# Backend field validation
+def field_validation(data):
+    if not data.get("title"):
```

Comment:
Can remove if we won't be using backend validation

Reply:
we can always use more validation!